packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -906,6 +900,22 @@ export function TransferPanelMain({
                     forToken={selectedToken}
                     prefix={selectedToken ? 'BALANCE: ' : ''}
                   />
+                  {app.isDepositMode &&
+                    isTokenMainnetUSDC(selectedToken?.address) && (
+                      <TokenBalance
+                        balance={
+                          erc20L2Balances?.[CommonAddress.ArbitrumOne.USDC] ??
+                          constants.Zero
+                        }
+                        on={NetworkType.l2}
+                        forToken={
+                          selectedToken
+                            ? { ...selectedToken, symbol: 'USDC' }
+                            : null
+                        }
+                        skipUSDCoverride
```

Comment:
Maybe we can generalize a bit and have a prop that we use to force a certain token symbol and skip the sanitization:

```suggestion
                        overrideTokenSymbol="USDC"
```